base/applications/network/arp/arp_msg.mc

```diff
@@ -81,6 +82,39 @@ Przykłady:
   > arp -s 157.55.85.212   00-aa-00-62-c6-09  .... Dodaje statyczny wpis.
   > arp -a                                    .... Wyświetla tabelę arp.
 .
+Language=Romanian
+Afișează și modifică tabelele de traducere din adrese IP în adrese fizice folosite de
+Protocol de rezolvare a adreselor (ARP).
+
+ARP -s inet_addr eth_addr [if_addr]
+ARP -d inet_addr [if_addr]
+ARP -a [inet_addr] [-N if_addr]
+
+  -a            Afișează intrările ARP actuale interogând datele protocolului
+                actual. Dacă inet_addr e specificat, adresele IP și fizică
+                sunt afișate numai pentru calculatorul specificat. Dacă
+                mai mult de o interfață de rețea folosește ARP, sunt afișate intrările
+                pentru fiecare tabel ARP.
+  -g            La fel ca la -a.
+  inet_addr     Specifică o adresă de internet.
+  -N if_addr    Afișează intrările ARP pentru interfața de rețea specificată
+                de if_addr.
+  -d            Șterge gazda specificată de inet_addr. inet_addr poate fi
+                marcat cu caracterul special * pentru a șterge toate gazdele.
+  -s            Adaugă gazda și asociază adresa de Internet inet_addr
+                cu adresa fizică eth_addr. Adresa fizică este
+                reprezentată ca 6 octeți hexazecimali separați de cratime. Intrarea
+                este permanentă.
+  eth_addr      Specifică adresa fizică.
+  if_addr       Dacă e prezent, acesta specifică adresa de Internet a
+            					interfeței al cărei tabel de traducere a adreselor ar trebui
+                modificat.
+                Dacă nu e prezentă, va fi folosită prima interfață
+                aplicabilă.
```

Comment:
This does not look good now. Please fix the indentation (i.e. the amount of spaces) before the word "interfeței", so that it does not stand out in relation to other lines. Also, since the contents of line №112 describe the same "if_addr" parameter, those can therefore be edited to continue at the end of the line above it (№111, the one with "modificat."), since there's plenty of space there, isn't it? Just keep lines under 80 characters, and use a bit of common sense.

Reply:
I understand now! Thank you!